internal/config/config.go

```diff
@@ -27,6 +27,15 @@ type MbopConfig struct {
 	IsInternalLabel        string
 	Debug                  bool
 
+	KeyCloakProtocol       string
```

Comment:
Do we need to think about these as two separate entities with regard to a) Keycloak (strictly for retrieving an access token from a service account) and b) the Keycloak user service (deployed separate from Keycloak as a wrapper api)?

Reply:
The host for each would be different, for instance. Unless maybe I'm missing something?

Reply:
Clarified that the user service implements the token url. 👍